src/gc/impl/conservative/gc.d

```diff
@@ -2854,6 +2854,16 @@ struct Gcx
 
     void scanBackground() nothrow
     {
+        version(linux) {
+            import core.sys.posix.signal;
+
+            // block all signals, so that scanThread will not receive a signal not intended for him.
+			// see https://issues.dlang.org/show_bug.cgi?id=20256
+            sigset_t new_mask;
+            sigfillset(&new_mask);
```

Comment:
Please make the indenting consistent with the code below.

Reply:
Sorry, should be ok now.

Reply:
This is still indented by two spaces. Also the please place the brace in `version(linux) {` on a new line.

Reply:
sorry for this stupid noise. Is it ok now?

Reply:
sorry, it was new unconfigured vim instance...